gui/window.go

```diff
@@ -781,6 +782,16 @@ func (window *Window) SetScanMedia(scannedMedia []*storage.BlockDevice) {
 	window.scanInfo.Media = scannedMedia
 }
 
+// Panic handles the gui crashes
+func (window *Window) Panic(err error) {
+	log.Debug("Panic")
+	displayErrorDialog(err)
```

Comment:
Should this move after the log.RequestCrashInfo()? To ensure we log the telemetry and the crash info before the user has a chance to dismiss and close the app?

Reply:
Done.